modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+:context: ossm-config-openshift-monitoring-only
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. First priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisite
+
+* {SMProductName} is installed.
+
+* `user-workload monitoring` is enabled. See "Enabling monitoring for user-defined projects".
+
+.Procedure
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+.Example `ServiceMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+    relabelings:
+    - action: replace
+      replacement: "basic-istio-system"
+      targetLabel: mesh_id
```

Comment:
Since we removed `mesh_id` from the `PodMonitor` you can remove it here as well.
```suggestion
    relabelings:
    - action: replace
      replacement: "basic-istio-system"
      targetLabel: mesh_id
```